### PR TITLE
fix (webapi): admin fixes

### LINKED ISSUE #295: Update Google Chrome Drivers for Selenium
2.2.5 is out: http://chromedriver.storage.googleapis.com/index.html?path=2.25/

With the current version of the drivers we have, the driver did not work on Ubuntu. With 2.2.5 it works fine.

### LINKED ISSUE #381: projects api endpoint not working
java.lang.RuntimeException: Cannot automatically determine case class field names and order for 'org.knora.webapi.messages.v1.responder.projectmessages.ProjectsResponseV1', please use the 'jsonFormat' overload with explicit field name specification

### LINKED ISSUE #382: password visible in userdate
session api

### LINKED ISSUE #383: GUI broken with new user and project management
The latest changes on `develop` require changes in the GUI as well.

This is my summary of things that have to be fixed:
- adapt login user name in browser tests
- adapt loading instructions for test data in browser tests
- adapt `jquery.resadd.js:1154`: setting the user's active project -> https://github.com/dhlab-basel/Knora/issues/118

Please run the browser tests!

### LINKED ISSUE #385: getting users profile not working
``http://localhost:3333/v1/users/username/root%40example.com``

### PR DESCRIPTION
**Pull request by subotic**
### Summary

Applying different bug fixes surrounding the recently added admin features.

### Issues
 - resolves #295
 - resolves #381
 - resolves #382 
 - resolves #385 
 - resolves #383

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
I figured out that the browser tests fail because data has been written back to `SALSAH.userdata` yet. So I added a function that accesses the window's JavaScript.

Please fix the other tests.

**Comment by tobiasschweizer** [comment]
Could you check if this works: Unable to run route http://localhost:3333/v1/resourcetypes?vocabulary=0

**Comment by loicjaouen** [comment]
underlying exception is:
```
org.knora.webapi.InconsistentTriplestoreDataException: Named graph not found: http://www.knora.org/ontology/dokubib
```

**Comment by loicjaouen** [comment]
if you comment out the `dokubib` section in `admin-data.ttl` or load `dokubib` onto, it should work better (at least removing it from admin-data helped here)

**Comment by tobiasschweizer** [comment]
@subotic Do the browser tests work for you now?

**Comment by tobiasschweizer** [comment]
Could you please fix the tests as soon as possible so that we can test this branch and merge it back to `develop`?

**Comment by subotic** [comment]
The Knora tests should pass now.

**Comment by tobiasschweizer** [comment]
Did you also try the browser tests?

**Comment by subotic** [comment]
The browser tests are still failing. I get exactly the same errors as already described in this comment https://github.com/dhlab-basel/Knora/issues/383#issuecomment-274846303

**Comment by tobiasschweizer** [comment]
Also with the latest change I made yesterday?

**Comment by tobiasschweizer** [comment]
> if you comment out the dokubib section in admin-data.ttl or load dokubib onto, it should work better (at least removing it from admin-data helped here)

**Comment by subotic** [comment]
We probably should then always load every ontology, for which we have a project defined inside ``admin-data.ttl``. I will add them all to ``app.triplestore.default-rdf-data`` inside ``application.conf``.

**Comment by tobiasschweizer** [comment]
ok, let me know if that solves the problem with the tests

**Comment by subotic** [comment]
Ok, now the browser tests are passing. I had to add vocabulary selection to some tests to select ``All``, as the default selected vocabulary was the one from the ``images`` project.

**Comment by tobiasschweizer** [comment]
This is very good! 
The tests also work for me now.

@subotic @benjamingeer Do you think we can merge this back to `develop`?

**Comment by benjamingeer** [comment]
If Tobias is happy, you can merge. :)

**Comment by tobiasschweizer** [comment]
I am very happy indeed :-)

But since I am not familiar with the details of user management, please have a look at the changes.

File: webapi/src/main/resources/application.conf [hunk 1]
```diff
@@ -274,8 +274,8 @@ app {
     }
 
     triplestore {
-        dbtype = "fuseki"
-        // dbtype = "graphdb"
+        // dbtype = "fuseki"
```
Comment (benjamingeer) [inline webapi/src/main/resources/application.conf]: Please leave Fuseki as the default.

Comment (tobiasschweizer) [inline webapi/src/main/resources/application.conf]: yes, I wondered about this, too :-)

@subotic ok, just this small thing and then you could merge it back.

Comment (subotic) [inline webapi/src/main/resources/application.conf]: it's changed

**Comment by subotic** [comment]
I'll wait for Travis to do it's thing and then I will merge the PR.

**Comment by tobiasschweizer** [comment]
Ok, great. Thanks.

**Pull request merged**